src/Navbar/test/NavbarStylesSpec.tsx

```diff
@@ -10,7 +10,7 @@ describe('Navbar styles', () => {
   it('Should render the correct styles', () => {
     const instanceRef = React.createRef<HTMLDivElement>();
     render(<Navbar ref={instanceRef} />);
-    const dom = getDOMNode(instanceRef.current);
+    const dom = instanceRef.current as Element;
     assert.equal(getStyle(dom, 'backgroundColor'), toRGB('#f7f7fa'), 'NavBar background-color');
```

Comment:
Use `expect` instead of `assert` as the test assertion to maintain code style consistency.